### PR TITLE
Removes scout api from compress base command

### PR DESCRIPTION
**Pull request by moonso**
This PR fixes a bug where the scout api was used when instantiating the meta compress class. When bam compression was removed the scout api was not necessary anymore but it was still being used in the cli base class

### How to test

Unfortunately there is no way to test fastq compression on stage since we point to production fastq files ...

### Expected PR outcome
Fastq compression should now be possible

## Review
- [x] code approved by HS
- [x] "Merge and deploy" approved by MM
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: cg/apps/crunchy/crunchy.py [hunk 1]
```diff
@@ -327,9 +328,12 @@ def get_sbatch_path(log_dir: Path, compression: str, run_name: str = None) -> Pa
 
     def _submit_sbatch(self, sbatch_content: str, sbatch_path: Path):
         """Submit SLURM job"""
+        LOG.info("Submit sbatch")
         if self.dry_run:
+            LOG.warning("HEJ")
```
Comment (henrikstranneheim) [inline cg/apps/crunchy/crunchy.py:333]:
```suggestion
```

File: cg/apps/crunchy/crunchy.py [hunk 2]
```diff
@@ -327,9 +328,12 @@ def get_sbatch_path(log_dir: Path, compression: str, run_name: str = None) -> Pa
...
             LOG.info("Would submit sbatch %s to slurm", sbatch_path)
             return
+        LOG.warning("DU")
```
Comment (henrikstranneheim) [inline cg/apps/crunchy/crunchy.py:336]:
```suggestion
```

**Review by henrikstranneheim — APPROVED** [review]
👍

**Comment by moonso** [comment]
Skärmavbild 2020-08-28 kl  10 14 58

Merged and deployed on hasta

**Pull request merged**